test/integration/patients/worklist/schedule.js

```diff
@@ -935,6 +948,11 @@ context('schedule page', function() {
       .find('.fa-square-check')
       .should('have.length', 0);
 
+    cy
+      .get('.schedule-list__table')
+      .find('.schedule-list__list-row .is-selected')
+      .should('have.length', 0);
+
```

Comment:
Verifies that the `.is-selected` class is removed for list items when the bulk selections are cleared/cancelled by the user.